app/src/main/java/io/github/droidkaigi/confsched/viewmodel/event/EventBus.java

```diff
@@ -0,0 +1,24 @@
+package io.github.droidkaigi.confsched.viewmodel.event;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+import rx.subjects.SerializedSubject;
+import rx.subjects.Subject;
+
+public class EventBus {
```

Comment:
I created Rx event bus class.
EventBus tends to be abused. It's really dangerous.
So my policy is that EventBus should be used to notify event to View from ViewModel, then package is `viewmodel/event/EventBus`.